src/lsm/cache_map.zig

```diff
@@ -157,9 +157,7 @@ pub fn CacheMapType(
                 } else {
                     // If it was an insert, append a tombstone to the scope rollback log.
                     const key = key_from_value(value);
-                    self.scope_rollback_log.appendAssumeCapacity(
-                        tombstone_from_key(key),
-                    );
+                    self.scope_rollback_log.appendAssumeCapacity(tombstone_from_key(key));
```

Comment:
FWIW, I slightly prefer the original formatting. The rule of thumb here is that I don't like code of the shape of `f(g(x))`, the composition of functions is easy to miss.  Though, I usually like to format those as

```
const y = g(x)
f(y)
```